tests/run-make/pgo-branch-weights/rmake.rs

```diff
@@ -0,0 +1,45 @@
+// This test generates an instrumented binary - a program which
+// will keep track of how many times it calls each function, a useful
+// feature for optimization. Then, an argument (aaaaaaaaaaaa2bbbbbbbbbbbb2bbbbbbbbbbbbbbbbcc)
+// is passed into the instrumented binary, which should react with a number of function calls
+// fully known in advance. (For example, the letter 'a' results in calling f1())
+
+// If the test passes, the expected function call count was added to the use-phase LLVM-IR.
+// See https://github.com/rust-lang/rust/pull/66631
+
+//@ needs-profiler-support
+//@ ignore-cross-compile
+
+// (This test has problems generating profdata on mingw. This could use further investigation.)
+//@ ignore-windows-gnu
```

Comment:
Suggestion: can you add a `FIXME` to this `ignore-windows-gnu` for https://github.com/rust-lang/rust/pull/122613?